### PR TITLE
Fix typo in "message too long" DM

### PR DESCRIPTION
**Pull request by clayboone**
I found a small bug in a message that most people will never see.

### CODE REVIEW

File: tests/test_commands_poms.py [hunk 1]
```diff
@@ -105,15 +107,37 @@ async def test_too_many_pom_descripts_causes_detailed_direct_message(self):
...
+            r"```fix\n"
+            r"\n"
+            r"The combined length of all the pom descriptions in your "
+            r"current session is longer than the maximum embed message field "
+            r"size for Discord embeds \([0-9]{4,}, Max is ([0-9]+)\)\. Please "
+            r"rename a few with \!(poms|bank)\.rename \(see \!help "
+            r"(poms|bank)\)\.```"
+        )
+
+        # FIXME: this should fails CI.
```
Comment (clayboone) [inline tests/test_commands_poms.py:125]: An addition to figure out CircleCI. To be removed shortly.

File: tests/test_commands_poms.py [hunk 2]
```diff
@@ -92,31 +93,76 @@ async def test_too_many_pom_descripts_causes_detailed_direct_message(self):
...
+        # The user was DM'd and only DM'd.
         self.assertTrue(self.ctx.author.send.called)
         self.assertFalse(any((self.ctx.send.called, self.ctx.reply.called)))
 
-        # All of the user's poms descripts are displayed in the DM's.
-        # NOTE: The first element in Mock.called_args_list is the one that
-        # caused the exception. Also, this will turn `_Call`'s into strings.
-        actual_combined_response = "\n".join(
-            (str(a) for a, _ in self.ctx.author.send.call_args_list[1:]))
+        first, second, third, *remainder = self.ctx.author.send.call_args_list
```
Comment (clayboone) [inline tests/test_commands_poms.py:132]: Thought I'd tackle this while we aren't merging this one yet.

Currently, the error messages get displayed to the user in a dynamic way depending on which part of the embed failed. This isn't quite the behavior I would expect, so this tests for a more conventional "errors always show up last" response.

`make test` should now fail in CI.

Comment (clayboone) [reply tests/test_commands_poms.py:133]: Fixed. CI should still fail with a few linter errors. @debashisbiswas

Comment (clayboone) [reply tests/test_commands_poms.py:133]: All good. I didn't leave in any FIXME's, so CI should be green now.

**Review by debashisbiswas — APPROVED** [review]
Looks good!

**Pull request merged**